src/Core/Grid/Definition/Factory/DeleteActionTrait.php

```diff
@@ -36,21 +36,35 @@
  */
 trait DeleteActionTrait
 {
-    protected function buildDeleteAction(string $deleteRouteName, string $deleteRouteParamName, string $deleteRouteParamField, string $method = 'POST'): RowActionInterface
-    {
+    protected function buildDeleteAction(
+        string $deleteRouteName,
+        string $deleteRouteParamName,
+        string $deleteRouteParamField,
+        string $method = 'POST',
+        array $extraRouteParams = [],
+        array $translations = []
+    ): RowActionInterface {
         return (new SubmitRowAction('delete'))
             ->setName($this->trans('Delete', [], 'Admin.Actions'))
             ->setIcon('delete')
             ->setOptions([
                 'route' => $deleteRouteName,
                 'route_param_name' => $deleteRouteParamName,
                 'route_param_field' => $deleteRouteParamField,
-                'confirm_message' => $this->trans('Are you sure you want to delete the selected item?', [], 'Admin.Notifications.Warning'),
-                'method' => $method,
+                'extra_route_params' => $extraRouteParams,
+                'confirm_message' => array_key_exists('confirm_message', $translations) ?
+                    $translations['confirm_message'] :
+                    $this->trans('Are you sure you want to delete the selected item(s)?', [], 'Admin.Global'),
```

Comment:
@LouiseBonnard The default message is this one
```$this->trans('Are you sure you want to delete the selected item(s)?', [], 'Admin.Global'),```

Reply:
```suggestion
                    $this->trans('Are you sure you want to delete the selected item(s)?', [], 'Admin.Notifications.Warning'),
```